src/drivers/mct/cime_config/config_compsets.xml

```diff
@@ -40,32 +41,32 @@
 
   <compset>
     <alias>A</alias>
-    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV</lname>
+    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV_SIAC</lname>
   </compset>
 
   <compset>
     <alias>ADSOM</alias>
-    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%SOM_DROF%NYF_SGLC_SWAV_TEST</lname>
+    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%SOM_DROF%NYF_SGLC_SWAV_SIAC_TEST</lname>
```

Comment:
Does anyone know what `_TEST` does in the compset name?
Any reason we shouldn't just get rid of it? @jedwards4b? @jgfouca?

Reply:
In some cases (I'm not sure about this one), `_TEST` reduces the required input data volume (e.g., using just one or a few years rather than the full time series).

Reply:
I was wondering about this because I cannot find any usage of the item. If it is used, I need to fix my latest PR (I did not know about this and testing did not catch it). If it is not used, I think we should get rid of cases that use it.

Reply:
@billsacks Is thinking of modifiers to the compset name like DTEST - that's different than a component called _TEST - as far as I know this isn't used anywhere in CIME.